diff -abru work/jboss-eap-6.1-src/pom.xml work/jboss-eap-6.1-src-work/pom.xml
--- work/jboss-eap-6.1-src/pom.xml	2013-05-09 13:06:56.000000000 +0200
+++ work/jboss-eap-6.1-src-work/pom.xml	2013-07-26 00:29:25.000000000 +0200
@@ -60,7 +60,7 @@
         <jboss.as.scm.connection>git@github.com:jbossas/jboss-as.git</jboss.as.scm.connection>
 
         <!-- Non-default plugin versions and configuration -->
-        <version.enforcer.plugin>1.2-redhat-1</version.enforcer.plugin>
+        <version.enforcer.plugin>1.2</version.enforcer.plugin>
         <version.jacoco.plugin>0.5.6.201201232323</version.jacoco.plugin> <!-- Needs to be set explicitely to match with org.jacoco:org.jacoco.ant - see testsuite. -->
 
         <!-- Surefire args -->
@@ -3445,7 +3445,7 @@
                 <groupId>org.jboss.byteman</groupId>
                 <artifactId>byteman-install</artifactId>
                 <scope>test</scope>
-                <version>${version.org.jboss.byteman}</version>
+                <version>2.0.1</version>
             </dependency>
 
             <dependency>
diff -abru work/jboss-eap-6.1-src/tools/maven/conf/settings.xml work/jboss-eap-6.1-src-work/tools/maven/conf/settings.xml
--- work/jboss-eap-6.1-src/tools/maven/conf/settings.xml	2013-05-09 13:06:56.000000000 +0200
+++ work/jboss-eap-6.1-src-work/tools/maven/conf/settings.xml	2013-07-26 21:55:58.000000000 +0200
@@ -1,132 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!--
-Licensed to the Apache Software Foundation (ASF) under one
-or more contributor license agreements.  See the NOTICE file
-distributed with this work for additional information
-regarding copyright ownership.  The ASF licenses this file
-to you under the Apache License, Version 2.0 (the
-"License"); you may not use this file except in compliance
-with the License.  You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing,
-software distributed under the License is distributed on an
-"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-KIND, either express or implied.  See the License for the
-specific language governing permissions and limitations
-under the License.
--->
-
-<!--
- | This is the configuration file for Maven. It can be specified at two levels:
- |
- |  1. User Level. This settings.xml file provides configuration for a single user, 
- |                 and is normally provided in ${user.home}/.m2/settings.xml.
- |
- |                 NOTE: This location can be overridden with the CLI option:
- |
- |                 -s /path/to/user/settings.xml
- |
- |  2. Global Level. This settings.xml file provides configuration for all Maven
- |                 users on a machine (assuming they're all using the same Maven
- |                 installation). It's normally provided in 
- |                 ${maven.home}/conf/settings.xml.
- |
- |                 NOTE: This location can be overridden with the CLI option:
- |
- |                 -gs /path/to/global/settings.xml
- |
- | The sections in this sample file are intended to give you a running start at
- | getting the most out of your Maven installation. Where appropriate, the default
- | values (values used when the setting is not specified) are provided.
- |
- |-->
 <settings xmlns="http://maven.apache.org/SETTINGS/1.0.0" 
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
           xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0 http://maven.apache.org/xsd/settings-1.0.0.xsd">
-  <!-- localRepository
-   | The path to the local repository maven will use to store artifacts.
-   |
-   | Default: ~/.m2/repository
-  <localRepository>/path/to/local/repo</localRepository>
-  -->
 
-  <!-- mirrors
-   | This is a list of mirrors to be used in downloading artifacts from remote repositories.
-   | 
-   | It works like this: a POM may declare a repository to use in resolving certain artifacts.
-   | However, this repository may have problems with heavy traffic at times, so people have mirrored
-   | it to several places.
-   |
-   | That repository definition will have a unique id, so we can create a mirror reference for that
-   | repository, to be used as an alternate download site. The mirror site will be the preferred 
-   | server for that repository.
-   |-->
-  <mirrors>
-    <!-- mirror
-     | Specifies a repository mirror site to use instead of a given repository. The repository that
-     | this mirror serves has an ID that matches the mirrorOf element of this mirror. IDs are used
-     | for inheritance and direct lookup purposes, and must be unique across the set of mirrors.
-     |
-    <mirror>
-      <id>mirrorId</id>
-      <mirrorOf>repositoryId</mirrorOf>
-      <name>Human Readable Name for this Mirror.</name>
-      <url>http://my.repository.com/repo/path</url>
-    </mirror>
-     -->
-
-    <!-- eap6 mirror settings
-     |   Mirror any repo found in pom files.  EAP builds must only pull from the mead-eap6 repos for building.
-     |   And may use the public JBoss repo and central for building and running tests.
-     -->
-    <mirror>
-      <id>eap6-mirror-setting</id>
-      <mirrorOf>
-         *,!central-eap6,!central-eap6-plugins,!jboss-public-eap6,!jboss-public-eap6-plugins
-      </mirrorOf>
-      <name>Mirror Settings for EAP 6 build</name>
-      <!-- Note: for rebuilds, this URL needs to be changed to the correct EAP 6 build Maven repo -->
-      <url>http://download.lab.bos.redhat.com/brewroot/repos/jb-eap-6-rhel-6-build/latest/maven/</url>
-    </mirror>
-  </mirrors>
-  
-  <!-- profiles
-   | This is a list of profiles which can be activated in a variety of ways, and which can modify
-   | the build process. Profiles provided in the settings.xml are intended to provide local machine-
-   | specific paths and repository locations which allow the build to work in the local environment.
-   |
-   | For example, if you have an integration testing plugin - like cactus - that needs to know where
-   | your Tomcat instance is installed, you can provide a variable here such that the variable is 
-   | dereferenced during the build process to configure the cactus plugin.
-   |
-   | As noted above, profiles can be activated in a variety of ways. One way - the activeProfiles
-   | section of this document (settings.xml) - will be discussed later. Another way essentially
-   | relies on the detection of a system property, either matching a particular value for the property,
-   | or merely testing its existence. Profiles can also be activated by JDK version prefix, where a 
-   | value of '1.4' might activate a profile when the build is executed on a JDK version of '1.4.2_07'.
-   | Finally, the list of active profiles can be specified directly from the command line.
-   |
-   | NOTE: For profiles defined in the settings.xml, you are restricted to specifying only artifact
-   |       repositories, plugin repositories, and free-form properties to be used as configuration
-   |       variables for plugins in the POM.
-   |
-   |-->
   <profiles>
  
-    <!-- The property "public-repos" can be used to activate public repos and public plugin repos -->
     <profile>
       <id>public-repositories</id>
       <activation>
         <property>
-          <name>public-repos</name>
+          <name>public-repositories</name>
         </property>
       </activation>
       <repositories>
         <repository>
-          <id>central-eap6</id>
+          <id>central</id>
           <name>Use unique IDs for downloading artifacts vs. plugins from central</name>
           <url>http://repo1.maven.org/maven2</url>
           <releases>
@@ -138,7 +27,7 @@
           </snapshots>
         </repository>
         <repository>
-          <id>jboss-public-eap6</id>
+          <id>jboss-public-repo</id>
           <name>JBoss Public Repository Group</name>
           <url>http://repository.jboss.org/nexus/content/groups/public/</url>
           <layout>default</layout>
@@ -154,104 +43,40 @@
       </repositories>
     </profile>
 
-    <!--  EAP 6 build repository settings
-     |    The EAP 6 Mead repositories are added with invalid URLs which will be replaced
-     |    by the mirror setting above.
-     |
-     |    The EAP Mead repository profile must be placed after public repository profile
-     |    but before public plugin repository profile.  This is a strange quirk in Maven
-     |    dependency resolution which causes dependency repositories to be added to the 
-     |    top of the list, but plugin repositories to the bottom of the list.
-     |    Putting this profile in between the other two allows the Mead repository 
-     |    to be prioritized over public repositories and public plugin repositories
-     |    when all repository profiles are active.
-     -->
+    <!-- Configure the JBoss EAP Maven repository -->
     <profile>
-      <id>eap-mead-repository</id>
-      <activation>
-        <property>
-          <name>!eap-mead-repository</name>
-        </property>
-      </activation>
+      <id>jboss-eap-maven-repository</id>
       <repositories>
         <repository>
-          <id>mead-eap6</id>
-          <name>MEAD EAP6 Maven Repository</name>
-          <url>ignore</url>
+          <id>jboss-eap</id>
+          <url>${env.EAP_REPO_URL}</url>
           <releases>
             <enabled>true</enabled>
-            <updatePolicy>never</updatePolicy>
           </releases>
           <snapshots>
             <enabled>false</enabled>
           </snapshots>
         </repository>
       </repositories>
-
-      <!-- Note: plugin repositories have non-matching IDs to dependency repositories.
-       |   This allows plugin repos to be mirrored separately.
-       -->
-      <pluginRepositories>
-        <pluginRepository>
-          <id>mead-eap6-plugins</id>
-          <name>MEAD EAP6 Maven Plugin Repository</name>
-          <url>ignore</url>
-          <releases>
-            <enabled>true</enabled>
-            <updatePolicy>never</updatePolicy>
-          </releases>
-          <snapshots>
-            <enabled>false</enabled>
-          </snapshots>
-        </pluginRepository>
-      </pluginRepositories>
-    </profile>
-
-    <profile>
-      <id>public-plugin-repositories</id>
-      <activation>
-        <property>
-          <name>public-repos</name>
-        </property>
-      </activation>
-      <!-- Note: plugin repositories have non-matching IDs to dependency repositories.
-       |   This allows plugin repos to be mirrored separately.
-       -->
       <pluginRepositories>
         <pluginRepository>
-          <id>central-eap6-plugins</id>
-          <name>Use unique IDs for downloading artifacts vs. plugins from central</name>
-          <url>http://repo1.maven.org/maven2</url>
+          <id>jboss-eap-maven-plugin-repository</id>
+          <url>${env.EAP_REPO_URL}</url>
           <releases>
             <enabled>true</enabled>
-            <updatePolicy>never</updatePolicy>
           </releases>
           <snapshots>
             <enabled>false</enabled>
           </snapshots>
         </pluginRepository>
-        <pluginRepository>
-          <id>jboss-public-eap6-plugins</id>
-          <name>JBoss Public Repository Group Plugins</name>
-          <url>http://repository.jboss.org/nexus/content/groups/public/</url>
-          <releases>
-            <enabled>true</enabled>
-          </releases>
-          <snapshots>
-            <enabled>true</enabled>
-          </snapshots>
-        </pluginRepository>
       </pluginRepositories>      
     </profile>
 
   </profiles>
 
-  <!-- activeProfiles
-   | List of profiles that are active for all builds.
-   |
   <activeProfiles>
-    <activeProfile>alwaysActiveProfile</activeProfile>
-    <activeProfile>anotherAlwaysActiveProfile</activeProfile>
+    <activeProfile>public-repositories</activeProfile>
+    <activeProfile>jboss-eap-maven-repository</activeProfile>
   </activeProfiles>
-  -->
+
 </settings>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/log4j/log4j/1.2.16-redhat-2/log4j-1.2.16-redhat-2.pom work/jboss-eap-6.1.0.GA-maven-repository-work/log4j/log4j/1.2.16-redhat-2/log4j-1.2.16-redhat-2.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/log4j/log4j/1.2.16-redhat-2/log4j-1.2.16-redhat-2.pom	2013-05-09 21:42:47.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/log4j/log4j/1.2.16-redhat-2/log4j-1.2.16-redhat-2.pom	2013-07-27 18:57:33.000000000 +0200
@@ -229,7 +229,7 @@
           <dependency>
             <groupId>org.apache.ant</groupId>
             <artifactId>ant-junit</artifactId>
-            <version>1.8.3-redhat-1</version>
+            <version>1.8.3</version>
           </dependency>
           <dependency>
             <groupId>junit</groupId>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/net/sf/ehcache/ehcache-core/2.5.1-redhat-1/ehcache-core-2.5.1-redhat-1.pom work/jboss-eap-6.1.0.GA-maven-repository-work/net/sf/ehcache/ehcache-core/2.5.1-redhat-1/ehcache-core-2.5.1-redhat-1.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/net/sf/ehcache/ehcache-core/2.5.1-redhat-1/ehcache-core-2.5.1-redhat-1.pom	2013-05-09 21:43:10.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/net/sf/ehcache/ehcache-core/2.5.1-redhat-1/ehcache-core-2.5.1-redhat-1.pom	2013-07-27 18:33:50.000000000 +0200
@@ -100,7 +100,7 @@
     <dependency>
       <groupId>org.hibernate</groupId>
       <artifactId>hibernate-core</artifactId>
-      <version>3.6.5.Final-redhat-2</version>
+      <version>3.6.5.Final</version>
       <scope>provided</scope>
     </dependency>
     <!--dependency>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/fusesource/jansi/jansi/1.9-redhat-1/jansi-1.9-redhat-1.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/fusesource/jansi/jansi/1.9-redhat-1/jansi-1.9-redhat-1.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/fusesource/jansi/jansi/1.9-redhat-1/jansi-1.9-redhat-1.pom	2013-05-09 21:42:50.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/fusesource/jansi/jansi/1.9-redhat-1/jansi-1.9-redhat-1.pom	2013-07-27 07:50:05.000000000 +0200
@@ -94,7 +94,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.10-redhat-2</version>
+      <version>4.10</version>
       <scope>test</scope>
       <exclusions>
         <exclusion>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/byteman/byteman/2.0.1-redhat-2/byteman-2.0.1-redhat-2.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/byteman/byteman/2.0.1-redhat-2/byteman-2.0.1-redhat-2.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/byteman/byteman/2.0.1-redhat-2/byteman-2.0.1-redhat-2.pom	2013-05-09 21:43:11.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/byteman/byteman/2.0.1-redhat-2/byteman-2.0.1-redhat-2.pom	2013-07-27 07:50:21.000000000 +0200
@@ -132,7 +132,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.10-redhat-2</version>
+      <version>4.10</version>
       <scope>test</scope>
       <exclusions>
         <exclusion>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/component/management/jboss-component-version-master/6.0.1-redhat-1/jboss-component-version-master-6.0.1-redhat-1.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/component/management/jboss-component-version-master/6.0.1-redhat-1/jboss-component-version-master-6.0.1-redhat-1.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/component/management/jboss-component-version-master/6.0.1-redhat-1/jboss-component-version-master-6.0.1-redhat-1.pom	2013-05-09 21:43:08.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/component/management/jboss-component-version-master/6.0.1-redhat-1/jboss-component-version-master-6.0.1-redhat-1.pom	2013-07-27 21:12:10.000000000 +0200
@@ -89,7 +89,7 @@
     <version.jaxen>1.1.3-redhat-2</version.jaxen>
     <version.jline>0.9.94-redhat-2</version.jline>
     <version.joda-time>1.6.2-redhat-3</version.joda-time>
-    <version.junit>4.10-redhat-2</version.junit>
+    <version.junit>4.10</version.junit>
     <version.log4j>1.2.16-redhat-2</version.log4j>
     <version.logkit>1.2</version.logkit> <!-- Unsupported -->
     <version.net.java.dev.msv>2011.1-redhat-2</version.net.java.dev.msv>
@@ -102,7 +102,7 @@
     <version.net.sourceforge.nekohtml>1.9.14-redhat-1</version.net.sourceforge.nekohtml>
     <version.nu.validator.htmlparser>1.0.7-redhat-2</version.nu.validator.htmlparser>
     <version.org.apache.abdera>1.1.3-redhat-1</version.org.apache.abdera>
-    <version.org.apache.ant>1.8.3-redhat-1</version.org.apache.ant>
+    <version.org.apache.ant>1.8.3</version.org.apache.ant>
     <version.org.apache.aries>0.3-redhat-2</version.org.apache.aries>
     <version.org.apache.aries.jmx>0.3-redhat-2</version.org.apache.aries.jmx>
     <version.org.apache.commons.commons-parent>22-redhat-2</version.org.apache.commons.commons-parent>
@@ -159,8 +159,8 @@
     <version.org.hibernate.javax.persistence.hibernate-jpa-2.0-api>1.0.1.Final-redhat-2</version.org.hibernate.javax.persistence.hibernate-jpa-2.0-api>
     <version.org.hibernate.validator>4.2.0.Final-redhat-2</version.org.hibernate.validator>
     <!-- Needed for hibernate 3 compat in AS/EAP -->
-    <version.org.hibernate3>3.6.5.Final-redhat-2</version.org.hibernate3>
-    <version.org.hibernate3.commons.annotations>3.2.0.Final-redhat-2</version.org.hibernate3.commons.annotations>
+    <version.org.hibernate3>3.6.5.Final</version.org.hibernate3>
+    <version.org.hibernate3.commons.annotations>3.2.0.Final</version.org.hibernate3.commons.annotations>
     <version.org.hornetq>2.2.23.Final-redhat-1</version.org.hornetq>
     <version.org.infinispan>5.1.8.Final-redhat-1</version.org.infinispan>
     <version.org.jacorb>2.3.2-redhat-2</version.org.jacorb>
@@ -173,7 +173,7 @@
     <!-- JBoss Web Native is downloaded and installed separately, so it is not in the Maven repository -->
     <!-- <version.org.jboss.as.jbossweb-native>2.0.10.Final-redhat-1</version.org.jboss.as.jbossweb-native> -->
     <version.org.jboss.ballroom>1.4.0.Final-redhat-1</version.org.jboss.ballroom>
-    <version.org.jboss.byteman>2.0.1-redhat-2</version.org.jboss.byteman>
+    <version.org.jboss.byteman>2.0.1</version.org.jboss.byteman>
     <version.org.jboss.classfilewriter>1.0.3.Final-redhat-1</version.org.jboss.classfilewriter>
     <version.org.jboss.com.sun.httpserver>1.0.1.Final-redhat-2</version.org.jboss.com.sun.httpserver>
     <version.org.jboss.common.jboss-common-beans>1.0.0.Final-redhat-2</version.org.jboss.common.jboss-common-beans>
@@ -183,7 +183,7 @@
     <version.org.jboss.interceptor>2.0.0.Final-redhat-1</version.org.jboss.interceptor>
     <version.org.jboss.invocation>1.1.1.Final-redhat-2</version.org.jboss.invocation>
     <version.org.jboss.ironjacamar>1.0.13.Final-redhat-1</version.org.jboss.ironjacamar>
-    <version.org.jboss.jandex>1.0.3.Final-redhat-2</version.org.jboss.jandex>
+    <version.org.jboss.jandex>1.0.3.Final</version.org.jboss.jandex>
     <version.org.jboss.jboss-common-core>2.2.17.GA-redhat-2</version.org.jboss.jboss-common-core>
     <version.org.jboss.jboss-dmr>1.1.1.Final-redhat-2</version.org.jboss.jboss-dmr>
     <version.org.jboss.jboss-transaction-spi>7.0.0.Final-redhat-2</version.org.jboss.jboss-transaction-spi>
@@ -295,7 +295,7 @@
     <version.org.picketbox>4.0.14.Final-redhat-2</version.org.picketbox>
     <version.org.picketbox.picketbox-commons>1.0.0.final-redhat-2</version.org.picketbox.picketbox-commons>
     <version.org.picketlink>2.1.3.1-redhat-1</version.org.picketlink>
-    <version.org.powermock>1.4.11-redhat-1</version.org.powermock>
+    <version.org.powermock>1.4.11</version.org.powermock>
     <version.org.projectodd.stilts>0.1.26-redhat-2</version.org.projectodd.stilts>
     <version.org.python.jython.standalone>2.5.2-redhat-2</version.org.python.jython.standalone>
     <version.org.rhq.helpers>3.0.4-redhat-2</version.org.rhq.helpers>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-1/jboss-component-version-master-6.1.0-redhat-1.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-1/jboss-component-version-master-6.1.0-redhat-1.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-1/jboss-component-version-master-6.1.0-redhat-1.pom	2013-05-09 21:43:08.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-1/jboss-component-version-master-6.1.0-redhat-1.pom	2013-07-27 21:12:18.000000000 +0200
@@ -91,7 +91,7 @@
     <version.jaxen>1.1.3-redhat-2</version.jaxen>
     <version.jline>0.9.94-redhat-2</version.jline>
     <version.joda-time>1.6.2-redhat-3</version.joda-time>
-    <version.junit>4.10-redhat-2</version.junit>
+    <version.junit>4.10</version.junit>
     <version.log4j>1.2.16-redhat-2</version.log4j>
     <version.logkit>1.2</version.logkit> <!-- Unsupported -->
     <version.net.java.dev.msv>2011.1-redhat-2</version.net.java.dev.msv>
@@ -104,7 +104,7 @@
     <version.net.sourceforge.nekohtml>1.9.14-redhat-1</version.net.sourceforge.nekohtml>
     <version.nu.validator.htmlparser>1.0.7-redhat-2</version.nu.validator.htmlparser>
     <version.org.apache.abdera>1.1.3-redhat-1</version.org.apache.abdera>
-    <version.org.apache.ant>1.8.3-redhat-1</version.org.apache.ant>
+    <version.org.apache.ant>1.8.3</version.org.apache.ant>
     <version.org.apache.aries>0.3-redhat-2</version.org.apache.aries>
     <version.org.apache.aries.jmx>0.3-redhat-2</version.org.apache.aries.jmx>
     <version.org.apache.commons.commons-parent>22-redhat-2</version.org.apache.commons.commons-parent>
@@ -134,7 +134,7 @@
     <version.org.apache.maven.wagon>1.0-beta-7</version.org.apache.maven.wagon>
     <version.org.apache.maven.wagon.http-lightweight>1.0</version.org.apache.maven.wagon.http-lightweight>
     <version.org.apache.mina>2.0.0-redhat-3</version.org.apache.mina>
-    <version.org.apache.myfaces.core>2.1.8-redhat-1</version.org.apache.myfaces.core>
+    <version.org.apache.myfaces.core>2.1.8</version.org.apache.myfaces.core>
     <version.org.apache.neethi>3.0.2-redhat-2</version.org.apache.neethi>
     <version.org.apache.santuario>1.5.3-redhat-1</version.org.apache.santuario>
     <version.org.apache.velocity>1.7-redhat-2</version.org.apache.velocity>
@@ -164,8 +164,8 @@
     <version.org.hibernate.javax.persistence.hibernate-jpa-2.0-api>1.0.1.Final-redhat-2</version.org.hibernate.javax.persistence.hibernate-jpa-2.0-api>
     <version.org.hibernate.validator>4.3.1.Final-redhat-1</version.org.hibernate.validator>
     <!-- Needed for hibernate 3 compat in AS/EAP -->
-    <version.org.hibernate3>3.6.5.Final-redhat-2</version.org.hibernate3>
-    <version.org.hibernate3.commons.annotations>3.2.0.Final-redhat-2</version.org.hibernate3.commons.annotations>
+    <version.org.hibernate3>3.6.5.Final</version.org.hibernate3>
+    <version.org.hibernate3.commons.annotations>3.2.0.Final</version.org.hibernate3.commons.annotations>
     <version.org.hornetq>2.3.0.CR2-redhat-1</version.org.hornetq>
     <version.org.infinispan>5.2.5.Final-redhat-1</version.org.infinispan>
     <version.org.jacorb>2.3.2-redhat-3</version.org.jacorb>
@@ -179,7 +179,7 @@
     <!-- JBoss Web Native is downloaded and installed separately, so it is not in the Maven repository -->
     <!-- <version.org.jboss.as.jbossweb-native>2.0.10.Final-redhat-1</version.org.jboss.as.jbossweb-native> -->
     <version.org.jboss.ballroom>1.5.2.Final-redhat-1</version.org.jboss.ballroom>
-    <version.org.jboss.byteman>2.0.1-redhat-2</version.org.jboss.byteman>
+    <version.org.jboss.byteman>2.0.1</version.org.jboss.byteman>
     <version.org.jboss.classfilewriter>1.0.3.Final-redhat-1</version.org.jboss.classfilewriter>
     <version.org.jboss.com.sun.httpserver>1.0.1.Final-redhat-2</version.org.jboss.com.sun.httpserver>
     <version.org.jboss.common.jboss-common-beans>1.1.0.Final-redhat-1</version.org.jboss.common.jboss-common-beans>
@@ -189,7 +189,7 @@
     <version.org.jboss.interceptor>2.0.0.Final-redhat-1</version.org.jboss.interceptor>
     <version.org.jboss.invocation>1.1.1.Final-redhat-2</version.org.jboss.invocation>
     <version.org.jboss.ironjacamar>1.0.16.Final-redhat-1</version.org.jboss.ironjacamar>
-    <version.org.jboss.jandex>1.0.3.Final-redhat-2</version.org.jboss.jandex>
+    <version.org.jboss.jandex>1.0.3.Final</version.org.jboss.jandex>
     <version.org.jboss.jboss-common-core>2.2.17.GA-redhat-2</version.org.jboss.jboss-common-core>
     <version.org.jboss.jboss-dmr>1.1.6.Final-redhat-1</version.org.jboss.jboss-dmr>
     <version.org.jboss.jboss-transaction-spi>7.0.0.Final-redhat-2</version.org.jboss.jboss-transaction-spi>
@@ -303,13 +303,13 @@
     <version.org.picketbox>4.0.16.Final-redhat-1</version.org.picketbox>
     <version.org.picketbox.picketbox-commons>1.0.0.final-redhat-2</version.org.picketbox.picketbox-commons>
     <version.org.picketlink>2.1.6.Final-redhat-1</version.org.picketlink>
-    <version.org.powermock>1.4.11-redhat-1</version.org.powermock>
+    <version.org.powermock>1.4.11</version.org.powermock>
     <version.org.projectodd.stilts>0.1.26-redhat-2</version.org.projectodd.stilts>
     <version.org.python.jython.standalone>2.5.2-redhat-2</version.org.python.jython.standalone>
     <version.org.rhq.helpers>3.0.4-redhat-2</version.org.rhq.helpers>
     <version.org.scala-lang>2.9.1.final-redhat-1</version.org.scala-lang>
     <version.org.scannotation>1.0.2-redhat-2</version.org.scannotation>
-    <version.org.slf4j>1.7.2-redhat-1</version.org.slf4j>
+    <version.org.slf4j>1.7.2</version.org.slf4j>
     <version.org.sonatype.aether>1.13.1</version.org.sonatype.aether>
     <version.org.sonatype.sisu>3.0.3</version.org.sonatype.sisu>
     <version.org.springframework>2.5.6.SEC03</version.org.springframework>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-2/jboss-component-version-master-6.1.0-redhat-2.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-2/jboss-component-version-master-6.1.0-redhat-2.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-2/jboss-component-version-master-6.1.0-redhat-2.pom	2013-05-09 21:43:08.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/component/management/jboss-component-version-master/6.1.0-redhat-2/jboss-component-version-master-6.1.0-redhat-2.pom	2013-07-27 21:12:25.000000000 +0200
@@ -91,7 +91,7 @@
     <version.jaxen>1.1.3-redhat-2</version.jaxen>
     <version.jline>0.9.94-redhat-2</version.jline>
     <version.joda-time>1.6.2-redhat-4</version.joda-time>
-    <version.junit>4.10-redhat-2</version.junit>
+    <version.junit>4.10</version.junit>
     <version.log4j>1.2.16-redhat-2</version.log4j>
     <version.logkit>1.2</version.logkit> <!-- Unsupported -->
     <version.net.java.dev.msv>2011.1-redhat-2</version.net.java.dev.msv>
@@ -104,7 +104,7 @@
     <version.net.sourceforge.nekohtml>1.9.14-redhat-1</version.net.sourceforge.nekohtml>
     <version.nu.validator.htmlparser>1.0.7-redhat-2</version.nu.validator.htmlparser>
     <version.org.apache.abdera>1.1.3-redhat-1</version.org.apache.abdera>
-    <version.org.apache.ant>1.8.3-redhat-1</version.org.apache.ant>
+    <version.org.apache.ant>1.8.3</version.org.apache.ant>
     <version.org.apache.aries>0.3-redhat-2</version.org.apache.aries>
     <version.org.apache.aries.jmx>0.3-redhat-2</version.org.apache.aries.jmx>
     <version.org.apache.commons.commons-parent>22-redhat-2</version.org.apache.commons.commons-parent>
@@ -134,7 +134,7 @@
     <version.org.apache.maven.wagon>1.0-beta-7</version.org.apache.maven.wagon>
     <version.org.apache.maven.wagon.http-lightweight>1.0</version.org.apache.maven.wagon.http-lightweight>
     <version.org.apache.mina>2.0.0-redhat-3</version.org.apache.mina>
-    <version.org.apache.myfaces.core>2.1.8-redhat-1</version.org.apache.myfaces.core>
+    <version.org.apache.myfaces.core>2.1.8</version.org.apache.myfaces.core>
     <version.org.apache.neethi>3.0.2-redhat-2</version.org.apache.neethi>
     <version.org.apache.santuario>1.5.3-redhat-1</version.org.apache.santuario>
     <version.org.apache.velocity>1.7-redhat-2</version.org.apache.velocity>
@@ -164,8 +164,8 @@
     <version.org.hibernate.javax.persistence.hibernate-jpa-2.0-api>1.0.1.Final-redhat-2</version.org.hibernate.javax.persistence.hibernate-jpa-2.0-api>
     <version.org.hibernate.validator>4.3.1.Final-redhat-1</version.org.hibernate.validator>
     <!-- Needed for hibernate 3 compat in AS/EAP -->
-    <version.org.hibernate3>3.6.5.Final-redhat-2</version.org.hibernate3>
-    <version.org.hibernate3.commons.annotations>3.2.0.Final-redhat-2</version.org.hibernate3.commons.annotations>
+    <version.org.hibernate3>3.6.5.Final</version.org.hibernate3>
+    <version.org.hibernate3.commons.annotations>3.2.0.Final</version.org.hibernate3.commons.annotations>
     <version.org.hornetq>2.3.1.Final-redhat-1</version.org.hornetq>
     <version.org.infinispan>5.2.6.Final-redhat-1</version.org.infinispan>
     <version.org.jacorb>2.3.2-redhat-4</version.org.jacorb>
@@ -179,7 +179,7 @@
     <!-- JBoss Web Native is downloaded and installed separately, so it is not in the Maven repository -->
     <!-- <version.org.jboss.as.jbossweb-native>2.0.10.Final-redhat-1</version.org.jboss.as.jbossweb-native> -->
     <version.org.jboss.ballroom>1.5.2.Final-redhat-1</version.org.jboss.ballroom>
-    <version.org.jboss.byteman>2.0.1-redhat-2</version.org.jboss.byteman>
+    <version.org.jboss.byteman>2.0.1</version.org.jboss.byteman>
     <version.org.jboss.classfilewriter>1.0.3.Final-redhat-1</version.org.jboss.classfilewriter>
     <version.org.jboss.com.sun.httpserver>1.0.1.Final-redhat-2</version.org.jboss.com.sun.httpserver>
     <version.org.jboss.common.jboss-common-beans>1.1.0.Final-redhat-1</version.org.jboss.common.jboss-common-beans>
@@ -189,7 +189,7 @@
     <version.org.jboss.interceptor>2.0.0.Final-redhat-1</version.org.jboss.interceptor>
     <version.org.jboss.invocation>1.1.1.Final-redhat-2</version.org.jboss.invocation>
     <version.org.jboss.ironjacamar>1.0.17.Final-redhat-1</version.org.jboss.ironjacamar>
-    <version.org.jboss.jandex>1.0.3.Final-redhat-2</version.org.jboss.jandex>
+    <version.org.jboss.jandex>1.0.3.Final</version.org.jboss.jandex>
     <version.org.jboss.jboss-common-core>2.2.17.GA-redhat-2</version.org.jboss.jboss-common-core>
     <version.org.jboss.jboss-dmr>1.1.6.Final-redhat-1</version.org.jboss.jboss-dmr>
     <version.org.jboss.jboss-transaction-spi>7.0.0.Final-redhat-2</version.org.jboss.jboss-transaction-spi>
@@ -303,13 +303,13 @@
     <version.org.picketbox>4.0.17.Final-redhat-1</version.org.picketbox>
     <version.org.picketbox.picketbox-commons>1.0.0.final-redhat-2</version.org.picketbox.picketbox-commons>
     <version.org.picketlink>2.1.6.Final-redhat-2</version.org.picketlink>
-    <version.org.powermock>1.4.11-redhat-1</version.org.powermock>
+    <version.org.powermock>1.4.11</version.org.powermock>
     <version.org.projectodd.stilts>0.1.26-redhat-2</version.org.projectodd.stilts>
     <version.org.python.jython.standalone>2.5.2-redhat-2</version.org.python.jython.standalone>
     <version.org.rhq.helpers>3.0.4-redhat-2</version.org.rhq.helpers>
     <version.org.scala-lang>2.9.1.final-redhat-1</version.org.scala-lang>
     <version.org.scannotation>1.0.2-redhat-2</version.org.scannotation>
-    <version.org.slf4j>1.7.2-redhat-1</version.org.slf4j>
+    <version.org.slf4j>1.7.2</version.org.slf4j>
     <version.org.sonatype.aether>1.13.1</version.org.sonatype.aether>
     <version.org.sonatype.sisu>3.0.3</version.org.sonatype.sisu>
     <version.org.springframework>2.5.6.SEC03</version.org.springframework>
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/jboss-parent/10-redhat-1/jboss-parent-10-redhat-1.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/jboss-parent/10-redhat-1/jboss-parent-10-redhat-1.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/jboss-parent/10-redhat-1/jboss-parent-10-redhat-1.pom	2013-05-09 21:43:08.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/jboss-parent/10-redhat-1/jboss-parent-10-redhat-1.pom	2013-07-26 22:02:50.000000000 +0200
@@ -72,89 +72,89 @@
     <!-- **************** -->
     <!-- Plugins versions -->
     <!-- **************** -->
-    <version.apache-rat.plugin>0.6-redhat-1</version.apache-rat.plugin>
-    <version.animal-sniffer.plugin>1.7-redhat-1</version.animal-sniffer.plugin>
-    <version.antlr.plugin>2.1-redhat-1</version.antlr.plugin>
-    <version.antrun.plugin>1.7-redhat-1</version.antrun.plugin>
-    <version.antrun-ext.plugin>1.8-redhat-1</version.antrun-ext.plugin>
-    <version.apt.plugin>1.0-alpha-4-redhat-1</version.apt.plugin>
-    <version.assembly.plugin>2.2.1-redhat-1</version.assembly.plugin>
-    <version.buildhelper.plugin>1.5-redhat-1</version.buildhelper.plugin>
-    <version.buildnumber.plugin>1.0-redhat-1</version.buildnumber.plugin>
-    <version.bundle.plugin>2.3.7-redhat-1</version.bundle.plugin>
-    <version.cal10n.plugin>0.7.3-redhat-1</version.cal10n.plugin>
-    <version.cargo.plugin>1.0.2-redhat-1</version.cargo.plugin>
-    <version.changes.plugin>2.5-redhat-1</version.changes.plugin>
-    <version.checkstyle.plugin>2.5-redhat-1</version.checkstyle.plugin>
-    <version.clean.plugin>2.4.1-redhat-1</version.clean.plugin>
-    <version.cli.plugin>0.6.3.CR2-redhat-1</version.cli.plugin>
-    <version.clirr.plugin>2.2.2-redhat-1</version.clirr.plugin>
-    <version.clover.plugin>2.4-redhat-1</version.clover.plugin>
-    <version.clover2.plugin>3.0.2-redhat-1</version.clover2.plugin>
-    <version.checkstyle.plugin>2.5-redhat-1</version.checkstyle.plugin>
-    <version.cobertura.plugin>2.5-redhat-1</version.cobertura.plugin>
-    <version.commons-build.plugin>1.3-redhat-1</version.commons-build.plugin>
-    <version.compiler.plugin>3.0-redhat-1</version.compiler.plugin>
-    <version.cxf.plugin>2.3.2-redhat-1</version.cxf.plugin>
-    <version.dependency.plugin>2.3-redhat-1</version.dependency.plugin>
-    <version.deploy.plugin>2.5-redhat-1</version.deploy.plugin>
-    <version.docbkx.plugin>2.0.9-redhat-1</version.docbkx.plugin>
-    <version.ear.plugin>2.4.2-redhat-1</version.ear.plugin>
-    <version.eclipse.plugin>2.8-redhat-1</version.eclipse.plugin>
-    <version.ejb.plugin>2.2.1-redhat-1</version.ejb.plugin>
-    <version.enforcer.plugin>1.0-redhat-1</version.enforcer.plugin>
-    <version.failsafe.plugin>2.11-redhat-1</version.failsafe.plugin>
-    <version.findbugs.plugin>2.2-redhat-1</version.findbugs.plugin>
-    <version.google-license.plugin>1.4.0-redhat-1</version.google-license.plugin>
-    <version.gpg.plugin>1.4-redhat-1</version.gpg.plugin>
-    <version.groovy.plugin>1.0-beta-1-redhat-1</version.groovy.plugin>
-    <version.ianal.plugin>1.0-alpha-1-redhat-1</version.ianal.plugin>
-    <version.idea.plugin>2.2-redhat-1</version.idea.plugin>
-    <version.injection.plugin>1.0.2-redhat-1</version.injection.plugin>
-    <version.install.plugin>2.3.1-redhat-1</version.install.plugin>
-    <version.jacoco.plugin>0.5.6.201201232323-redhat-1</version.jacoco.plugin>
-    <version.jar.plugin>2.3.1-redhat-1</version.jar.plugin>
-    <version.javacc.plugin>2.6-redhat-1</version.javacc.plugin>
-    <version.javadoc.plugin>2.8-redhat-1</version.javadoc.plugin>
-    <version.javancss.plugin>2.0-redhat-1</version.javancss.plugin>
-    <version.jaxb2.plugin>1.3-redhat-1</version.jaxb2.plugin>
-    <version.jboss-packaging.plugin>2.1-redhat-1</version.jboss-packaging.plugin>
-    <version.jdiff.plugin>1.5-redhat-1</version.jdiff.plugin>
-    <version.jdocbook.plugin>2.2.3-redhat-1</version.jdocbook.plugin>
-    <version.jdocbook-style.plugin>2.0.0-redhat-1</version.jdocbook-style.plugin>
-    <version.jdepend.plugin>2.0-beta-2-redhat-1</version.jdepend.plugin>
-    <version.jxr.plugin>2.2-redhat-1</version.jxr.plugin>
-    <version.license.plugin>1.0-redhat-1</version.license.plugin>
-    <version.mycila-license.plugin>1.5.0-redhat-1</version.mycila-license.plugin>
-    <version.maven.plugin>1.2-redhat-1</version.maven.plugin>
-    <version.patch.plugin>1.1.1-redhat-1</version.patch.plugin>
-    <version.pir.plugin>2.2-redhat-1</version.pir.plugin><!-- maven-project-info-reports-plugins -->
-    <version.plugin.plugin>2.9-redhat-1</version.plugin.plugin>
-    <version.pmd.plugin>2.5-redhat-1</version.pmd.plugin>
-    <version.properties.plugin>1.0-alpha-2-redhat-1</version.properties.plugin>
-    <version.rar.plugin>2.2-redhat-1</version.rar.plugin>
-    <version.rat.plugin>1.0-alpha-3-redhat-1</version.rat.plugin>
-    <version.release.plugin>2.1-redhat-1</version.release.plugin>
-    <version.remote-resources.plugin>1.1-redhat-1</version.remote-resources.plugin>
-    <version.resources.plugin>2.5-redhat-1</version.resources.plugin>
-    <version.retrotranslator.plugin>1.0-alpha-4-redhat-1</version.retrotranslator.plugin>
-    <version.rmic.plugin>1.0-redhat-1</version.rmic.plugin>
-    <version.scala.plugin>2.15.2-redhat-1</version.scala.plugin>
-    <version.scm.plugin>1.5-redhat-1</version.scm.plugin>
-    <version.shade.plugin>1.5-redhat-1</version.shade.plugin>
-    <version.site.plugin>3.0-beta-3-redhat-1</version.site.plugin>
-    <version.sonar.plugin>1.0-beta-1-redhat-1</version.sonar.plugin>
-    <version.source.plugin>2.1.2-redhat-1</version.source.plugin>
-    <version.surefire.plugin>2.11-redhat-1</version.surefire.plugin>
-    <version.taglist.plugin>2.4-redhat-1</version.taglist.plugin>
-    <version.timestamp.plugin>1.0-redhat-1</version.timestamp.plugin>
-    <version.tools.plugin>1.2-redhat-1</version.tools.plugin>
-    <version.unsigner.plugin>0.2-redhat-1</version.unsigner.plugin>
-    <version.versions.plugin>1.1-redhat-1</version.versions.plugin>
-    <version.war.plugin>2.1.1-redhat-1</version.war.plugin>
-    <version.xdoc.plugin>1.9.2-redhat-1</version.xdoc.plugin>
-    <version.xml.plugin>1.0-redhat-1</version.xml.plugin>
-    <version.zanata.plugin>1.5.0-redhat-1</version.zanata.plugin>
+    <version.apache-rat.plugin>0.6</version.apache-rat.plugin>
+    <version.animal-sniffer.plugin>1.7</version.animal-sniffer.plugin>
+    <version.antlr.plugin>2.1</version.antlr.plugin>
+    <version.antrun.plugin>1.7</version.antrun.plugin>
+    <version.antrun-ext.plugin>1.8</version.antrun-ext.plugin>
+    <version.apt.plugin>1.0-alpha-4</version.apt.plugin>
+    <version.assembly.plugin>2.2.1</version.assembly.plugin>
+    <version.buildhelper.plugin>1.5</version.buildhelper.plugin>
+    <version.buildnumber.plugin>1.0</version.buildnumber.plugin>
+    <version.bundle.plugin>2.3.7</version.bundle.plugin>
+    <version.cal10n.plugin>0.7.3</version.cal10n.plugin>
+    <version.cargo.plugin>1.0.2</version.cargo.plugin>
+    <version.changes.plugin>2.5</version.changes.plugin>
+    <version.checkstyle.plugin>2.5</version.checkstyle.plugin>
+    <version.clean.plugin>2.4.1</version.clean.plugin>
+    <version.cli.plugin>0.6.3.CR2</version.cli.plugin>
+    <version.clirr.plugin>2.2.2</version.clirr.plugin>
+    <version.clover.plugin>2.4</version.clover.plugin>
+    <version.clover2.plugin>3.0.2</version.clover2.plugin>
+    <version.checkstyle.plugin>2.5</version.checkstyle.plugin>
+    <version.cobertura.plugin>2.5</version.cobertura.plugin>
+    <version.commons-build.plugin>1.3</version.commons-build.plugin>
+    <version.compiler.plugin>3.0</version.compiler.plugin>
+    <version.cxf.plugin>2.3.2</version.cxf.plugin>
+    <version.dependency.plugin>2.3</version.dependency.plugin>
+    <version.deploy.plugin>2.5</version.deploy.plugin>
+    <version.docbkx.plugin>2.0.9</version.docbkx.plugin>
+    <version.ear.plugin>2.4.2</version.ear.plugin>
+    <version.eclipse.plugin>2.8</version.eclipse.plugin>
+    <version.ejb.plugin>2.2.1</version.ejb.plugin>
+    <version.enforcer.plugin>1.0</version.enforcer.plugin>
+    <version.failsafe.plugin>2.11</version.failsafe.plugin>
+    <version.findbugs.plugin>2.2</version.findbugs.plugin>
+    <version.google-license.plugin>1.4.0</version.google-license.plugin>
+    <version.gpg.plugin>1.4</version.gpg.plugin>
+    <version.groovy.plugin>1.0-beta-1</version.groovy.plugin>
+    <version.ianal.plugin>1.0-alpha-1</version.ianal.plugin>
+    <version.idea.plugin>2.2</version.idea.plugin>
+    <version.injection.plugin>1.0.2</version.injection.plugin>
+    <version.install.plugin>2.3.1</version.install.plugin>
+    <version.jacoco.plugin>0.5.6.201201232323</version.jacoco.plugin>
+    <version.jar.plugin>2.3.1</version.jar.plugin>
+    <version.javacc.plugin>2.6</version.javacc.plugin>
+    <version.javadoc.plugin>2.8</version.javadoc.plugin>
+    <version.javancss.plugin>2.0</version.javancss.plugin>
+    <version.jaxb2.plugin>1.3</version.jaxb2.plugin>
+    <version.jboss-packaging.plugin>2.1</version.jboss-packaging.plugin>
+    <version.jdiff.plugin>1.5</version.jdiff.plugin>
+    <version.jdocbook.plugin>2.2.3</version.jdocbook.plugin>
+    <version.jdocbook-style.plugin>2.0.0</version.jdocbook-style.plugin>
+    <version.jdepend.plugin>2.0-beta-2</version.jdepend.plugin>
+    <version.jxr.plugin>2.2</version.jxr.plugin>
+    <version.license.plugin>1.0</version.license.plugin>
+    <version.mycila-license.plugin>1.5.0</version.mycila-license.plugin>
+    <version.maven.plugin>1.2</version.maven.plugin>
+    <version.patch.plugin>1.1.1</version.patch.plugin>
+    <version.pir.plugin>2.2</version.pir.plugin><!-- maven-project-info-reports-plugins -->
+    <version.plugin.plugin>2.9</version.plugin.plugin>
+    <version.pmd.plugin>2.5</version.pmd.plugin>
+    <version.properties.plugin>1.0-alpha-2</version.properties.plugin>
+    <version.rar.plugin>2.2</version.rar.plugin>
+    <version.rat.plugin>1.0-alpha-3</version.rat.plugin>
+    <version.release.plugin>2.1</version.release.plugin>
+    <version.remote-resources.plugin>1.1</version.remote-resources.plugin>
+    <version.resources.plugin>2.5</version.resources.plugin>
+    <version.retrotranslator.plugin>1.0-alpha-4</version.retrotranslator.plugin>
+    <version.rmic.plugin>1.0</version.rmic.plugin>
+    <version.scala.plugin>2.15.2</version.scala.plugin>
+    <version.scm.plugin>1.5</version.scm.plugin>
+    <version.shade.plugin>2.0</version.shade.plugin>
+    <version.site.plugin>3.0-beta-3</version.site.plugin>
+    <version.sonar.plugin>1.0-beta-1</version.sonar.plugin>
+    <version.source.plugin>2.1.2</version.source.plugin>
+    <version.surefire.plugin>2.11</version.surefire.plugin>
+    <version.taglist.plugin>2.4</version.taglist.plugin>
+    <version.timestamp.plugin>1.0</version.timestamp.plugin>
+    <version.tools.plugin>1.2</version.tools.plugin>
+    <version.unsigner.plugin>0.2</version.unsigner.plugin>
+    <version.versions.plugin>1.1</version.versions.plugin>
+    <version.war.plugin>2.1.1</version.war.plugin>
+    <version.xdoc.plugin>1.9.2</version.xdoc.plugin>
+    <version.xml.plugin>1.0</version.xml.plugin>
+    <version.zanata.plugin>1.5.0</version.zanata.plugin>
 
     <!-- ***************** -->
     <!-- Repository Deployment URLs -->
diff -abru work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/jboss-parent/10-redhat-2/jboss-parent-10-redhat-2.pom work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/jboss-parent/10-redhat-2/jboss-parent-10-redhat-2.pom
--- work/jboss-eap-6.1.0.GA-maven-repository/org/jboss/jboss-parent/10-redhat-2/jboss-parent-10-redhat-2.pom	2013-05-09 21:43:08.000000000 +0200
+++ work/jboss-eap-6.1.0.GA-maven-repository-work/org/jboss/jboss-parent/10-redhat-2/jboss-parent-10-redhat-2.pom	2013-07-26 22:20:54.000000000 +0200
@@ -72,90 +72,90 @@
     <!-- **************** -->
     <!-- Plugins versions -->
     <!-- **************** -->
-    <version.apache-rat.plugin>0.6-redhat-1</version.apache-rat.plugin>
-    <version.animal-sniffer.plugin>1.7-redhat-1</version.animal-sniffer.plugin>
-    <version.antlr.plugin>2.1-redhat-1</version.antlr.plugin>
-    <version.antrun.plugin>1.7-redhat-1</version.antrun.plugin>
-    <version.antrun-ext.plugin>1.8-redhat-1</version.antrun-ext.plugin>
-    <version.apt.plugin>1.0-alpha-4-redhat-1</version.apt.plugin>
-    <version.assembly.plugin>2.2.1-redhat-1</version.assembly.plugin>
-    <version.buildhelper.plugin>1.5-redhat-1</version.buildhelper.plugin>
-    <version.buildnumber.plugin>1.0-redhat-1</version.buildnumber.plugin>
-    <version.bundle.plugin>2.3.7-redhat-1</version.bundle.plugin>
-    <version.cal10n.plugin>0.7.3-redhat-1</version.cal10n.plugin>
-    <version.cargo.plugin>1.0.2-redhat-1</version.cargo.plugin>
-    <version.changes.plugin>2.5-redhat-1</version.changes.plugin>
-    <version.checkstyle.plugin>2.5-redhat-1</version.checkstyle.plugin>
-    <version.clean.plugin>2.4.1-redhat-1</version.clean.plugin>
-    <version.cli.plugin>0.6.3.CR2-redhat-1</version.cli.plugin>
-    <version.clirr.plugin>2.2.2-redhat-1</version.clirr.plugin>
-    <version.clover.plugin>2.4-redhat-1</version.clover.plugin>
-    <version.clover2.plugin>3.0.2-redhat-1</version.clover2.plugin>
-    <version.checkstyle.plugin>2.5-redhat-1</version.checkstyle.plugin>
-    <version.cobertura.plugin>2.5-redhat-1</version.cobertura.plugin>
-    <version.commons-build.plugin>1.3-redhat-1</version.commons-build.plugin>
-    <version.compiler.plugin>3.0-redhat-1</version.compiler.plugin>
-    <version.cxf.plugin>2.3.2-redhat-1</version.cxf.plugin>
-    <version.dependency.plugin>2.3-redhat-1</version.dependency.plugin>
-    <version.deploy.plugin>2.5-redhat-1</version.deploy.plugin>
-    <version.docbkx.plugin>2.0.9-redhat-1</version.docbkx.plugin>
-    <version.ear.plugin>2.4.2-redhat-1</version.ear.plugin>
-    <version.eclipse.plugin>2.8-redhat-1</version.eclipse.plugin>
-    <version.ejb.plugin>2.2.1-redhat-1</version.ejb.plugin>
-    <version.enforcer.plugin>1.0-redhat-1</version.enforcer.plugin>
-    <version.failsafe.plugin>2.11-redhat-1</version.failsafe.plugin>
-    <version.findbugs.plugin>2.2-redhat-1</version.findbugs.plugin>
-    <version.google-license.plugin>1.4.0-redhat-1</version.google-license.plugin>
-    <version.gpg.plugin>1.4-redhat-1</version.gpg.plugin>
-    <version.groovy.plugin>1.0-beta-1-redhat-1</version.groovy.plugin>
-    <version.ianal.plugin>1.0-alpha-1-redhat-1</version.ianal.plugin>
-    <version.idea.plugin>2.2-redhat-1</version.idea.plugin>
-    <version.idlj.plugin>1.2.1-redhat-1</version.idlj.plugin>
-    <version.injection.plugin>1.0.2-redhat-1</version.injection.plugin>
-    <version.install.plugin>2.3.1-redhat-1</version.install.plugin>
-    <version.jacoco.plugin>0.5.6.201201232323-redhat-1</version.jacoco.plugin>
-    <version.jar.plugin>2.3.1-redhat-1</version.jar.plugin>
-    <version.javacc.plugin>2.6-redhat-1</version.javacc.plugin>
-    <version.javadoc.plugin>2.8-redhat-1</version.javadoc.plugin>
-    <version.javancss.plugin>2.0-redhat-1</version.javancss.plugin>
-    <version.jaxb2.plugin>1.3-redhat-1</version.jaxb2.plugin>
-    <version.jboss-packaging.plugin>2.1-redhat-1</version.jboss-packaging.plugin>
-    <version.jdiff.plugin>1.5-redhat-1</version.jdiff.plugin>
-    <version.jdocbook.plugin>2.2.3-redhat-1</version.jdocbook.plugin>
-    <version.jdocbook-style.plugin>2.0.0-redhat-1</version.jdocbook-style.plugin>
-    <version.jdepend.plugin>2.0-beta-2-redhat-1</version.jdepend.plugin>
-    <version.jxr.plugin>2.2-redhat-1</version.jxr.plugin>
-    <version.license.plugin>1.0-redhat-1</version.license.plugin>
-    <version.mycila-license.plugin>1.5.0-redhat-1</version.mycila-license.plugin>
-    <version.maven.plugin>1.2-redhat-1</version.maven.plugin>
-    <version.patch.plugin>1.1.1-redhat-1</version.patch.plugin>
-    <version.pir.plugin>2.2-redhat-1</version.pir.plugin><!-- maven-project-info-reports-plugins -->
-    <version.plugin.plugin>2.9-redhat-1</version.plugin.plugin>
-    <version.pmd.plugin>2.5-redhat-1</version.pmd.plugin>
-    <version.properties.plugin>1.0-alpha-2-redhat-1</version.properties.plugin>
-    <version.rar.plugin>2.2-redhat-1</version.rar.plugin>
-    <version.rat.plugin>1.0-alpha-3-redhat-1</version.rat.plugin>
-    <version.release.plugin>2.1-redhat-1</version.release.plugin>
-    <version.remote-resources.plugin>1.1-redhat-1</version.remote-resources.plugin>
-    <version.resources.plugin>2.5-redhat-1</version.resources.plugin>
-    <version.retrotranslator.plugin>1.0-alpha-4-redhat-1</version.retrotranslator.plugin>
-    <version.rmic.plugin>1.0-redhat-1</version.rmic.plugin>
-    <version.scala.plugin>2.15.2-redhat-1</version.scala.plugin>
-    <version.scm.plugin>1.5-redhat-1</version.scm.plugin>
-    <version.shade.plugin>1.5-redhat-1</version.shade.plugin>
-    <version.site.plugin>3.0-beta-3-redhat-1</version.site.plugin>
-    <version.sonar.plugin>1.0-beta-1-redhat-1</version.sonar.plugin>
-    <version.source.plugin>2.1.2-redhat-1</version.source.plugin>
-    <version.surefire.plugin>2.11-redhat-1</version.surefire.plugin>
-    <version.taglist.plugin>2.4-redhat-1</version.taglist.plugin>
-    <version.timestamp.plugin>1.0-redhat-1</version.timestamp.plugin>
-    <version.tools.plugin>1.2-redhat-1</version.tools.plugin>
-    <version.unsigner.plugin>0.2-redhat-1</version.unsigner.plugin>
-    <version.versions.plugin>1.1-redhat-1</version.versions.plugin>
-    <version.war.plugin>2.1.1-redhat-1</version.war.plugin>
-    <version.xdoc.plugin>1.9.2-redhat-1</version.xdoc.plugin>
-    <version.xml.plugin>1.0-redhat-1</version.xml.plugin>
-    <version.zanata.plugin>1.5.0-redhat-1</version.zanata.plugin>
+    <version.apache-rat.plugin>0.6</version.apache-rat.plugin>
+    <version.animal-sniffer.plugin>1.7</version.animal-sniffer.plugin>
+    <version.antlr.plugin>2.1</version.antlr.plugin>
+    <version.antrun.plugin>1.7</version.antrun.plugin>
+    <version.antrun-ext.plugin>1.8</version.antrun-ext.plugin>
+    <version.apt.plugin>1.0-alpha-4</version.apt.plugin>
+    <version.assembly.plugin>2.2.1</version.assembly.plugin>
+    <version.buildhelper.plugin>1.5</version.buildhelper.plugin>
+    <version.buildnumber.plugin>1.0</version.buildnumber.plugin>
+    <version.bundle.plugin>2.3.7</version.bundle.plugin>
+    <version.cal10n.plugin>0.7.3</version.cal10n.plugin>
+    <version.cargo.plugin>1.0.2</version.cargo.plugin>
+    <version.changes.plugin>2.5</version.changes.plugin>
+    <version.checkstyle.plugin>2.5</version.checkstyle.plugin>
+    <version.clean.plugin>2.4.1</version.clean.plugin>
+    <version.cli.plugin>0.6.3.CR2</version.cli.plugin>
+    <version.clirr.plugin>2.2.2</version.clirr.plugin>
+    <version.clover.plugin>2.4</version.clover.plugin>
+    <version.clover2.plugin>3.0.2</version.clover2.plugin>
+    <version.checkstyle.plugin>2.5</version.checkstyle.plugin>
+    <version.cobertura.plugin>2.5</version.cobertura.plugin>
+    <version.commons-build.plugin>1.3</version.commons-build.plugin>
+    <version.compiler.plugin>3.0</version.compiler.plugin>
+    <version.cxf.plugin>2.3.2</version.cxf.plugin>
+    <version.dependency.plugin>2.3</version.dependency.plugin>
+    <version.deploy.plugin>2.5</version.deploy.plugin>
+    <version.docbkx.plugin>2.0.9</version.docbkx.plugin>
+    <version.ear.plugin>2.4.2</version.ear.plugin>
+    <version.eclipse.plugin>2.8</version.eclipse.plugin>
+    <version.ejb.plugin>2.2.1</version.ejb.plugin>
+    <version.enforcer.plugin>1.0</version.enforcer.plugin>
+    <version.failsafe.plugin>2.11</version.failsafe.plugin>
+    <version.findbugs.plugin>2.2</version.findbugs.plugin>
+    <version.google-license.plugin>1.4.0</version.google-license.plugin>
+    <version.gpg.plugin>1.4</version.gpg.plugin>
+    <version.groovy.plugin>1.0-beta-1</version.groovy.plugin>
+    <version.ianal.plugin>1.0-alpha-1</version.ianal.plugin>
+    <version.idea.plugin>2.2</version.idea.plugin>
+    <version.idlj.plugin>1.2.1</version.idlj.plugin>
+    <version.injection.plugin>1.0.2</version.injection.plugin>
+    <version.install.plugin>2.3.1</version.install.plugin>
+    <version.jacoco.plugin>0.5.6.201201232323</version.jacoco.plugin>
+    <version.jar.plugin>2.3.1</version.jar.plugin>
+    <version.javacc.plugin>2.6</version.javacc.plugin>
+    <version.javadoc.plugin>2.8</version.javadoc.plugin>
+    <version.javancss.plugin>2.0</version.javancss.plugin>
+    <version.jaxb2.plugin>1.3</version.jaxb2.plugin>
+    <version.jboss-packaging.plugin>2.1</version.jboss-packaging.plugin>
+    <version.jdiff.plugin>1.5</version.jdiff.plugin>
+    <version.jdocbook.plugin>2.2.3</version.jdocbook.plugin>
+    <version.jdocbook-style.plugin>2.0.0</version.jdocbook-style.plugin>
+    <version.jdepend.plugin>2.0-beta-2</version.jdepend.plugin>
+    <version.jxr.plugin>2.2</version.jxr.plugin>
+    <version.license.plugin>1.0</version.license.plugin>
+    <version.mycila-license.plugin>1.5.0</version.mycila-license.plugin>
+    <version.maven.plugin>1.2</version.maven.plugin>
+    <version.patch.plugin>1.1.1</version.patch.plugin>
+    <version.pir.plugin>2.2</version.pir.plugin><!-- maven-project-info-reports-plugins -->
+    <version.plugin.plugin>2.9</version.plugin.plugin>
+    <version.pmd.plugin>2.5</version.pmd.plugin>
+    <version.properties.plugin>1.0-alpha-2</version.properties.plugin>
+    <version.rar.plugin>2.2</version.rar.plugin>
+    <version.rat.plugin>1.0-alpha-3</version.rat.plugin>
+    <version.release.plugin>2.1</version.release.plugin>
+    <version.remote-resources.plugin>1.1</version.remote-resources.plugin>
+    <version.resources.plugin>2.5</version.resources.plugin>
+    <version.retrotranslator.plugin>1.0-alpha-4</version.retrotranslator.plugin>
+    <version.rmic.plugin>1.0</version.rmic.plugin>
+    <version.scala.plugin>2.15.2</version.scala.plugin>
+    <version.scm.plugin>1.5</version.scm.plugin>
+    <version.shade.plugin>2.0</version.shade.plugin>
+    <version.site.plugin>3.0-beta-3</version.site.plugin>
+    <version.sonar.plugin>1.0-beta-1</version.sonar.plugin>
+    <version.source.plugin>2.1.2</version.source.plugin>
+    <version.surefire.plugin>2.11</version.surefire.plugin>
+    <version.taglist.plugin>2.4</version.taglist.plugin>
+    <version.timestamp.plugin>1.0</version.timestamp.plugin>
+    <version.tools.plugin>1.2</version.tools.plugin>
+    <version.unsigner.plugin>0.2</version.unsigner.plugin>
+    <version.versions.plugin>1.1</version.versions.plugin>
+    <version.war.plugin>2.1.1</version.war.plugin>
+    <version.xdoc.plugin>1.9.2</version.xdoc.plugin>
+    <version.xml.plugin>1.0</version.xml.plugin>
+    <version.zanata.plugin>1.5.0</version.zanata.plugin>
 
     <!-- ***************** -->
     <!-- Repository Deployment URLs -->
